libs/remote_storage/src/local_fs.rs

```diff
@@ -660,6 +662,22 @@ mod fs_tests {
             "Second part bytes should be returned when requested"
         );
 
+        let suffix_bytes = storage
+            .download_byte_range(&upload_target, 13, None)
+            .await?
+            .download_stream;
+        let suffix_bytes = aggregate(suffix_bytes).await?;
+        let suffix = std::str::from_utf8(&suffix_bytes)?;
+        assert_eq!(upload_name, suffix);
+
+        let all_bytes = storage
+            .download_byte_range(&upload_target, 0, None)
+            .await?
+            .download_stream;
+        let all_bytes = aggregate(all_bytes).await?;
+        let all_bytes = std::str::from_utf8(&all_bytes)?;
+        assert_eq!("contents for upload_1", all_bytes);
```

Comment:
nit: could you use `LocalFs::dummy_contents(upload_name)`? You'd have to make it public first.

Reply:
It would be visible all right; I prefer to see constants in tests, but might as well.
```suggestion
        assert_eq!(dummy_contents("upload_1"), all_bytes);
```

Reply:
dried up in 411840815